webclient/index.html

```diff
@@ -681,6 +685,35 @@
         link.click();
     }
 
+
+
+    const configureAuthClient = async ()=>{
+
+        const config = {domain: "{{ .Env.AUTH0_DOMAIN }}", clientId: "{{ .Env.AUTH0_CLIENT_ID }}"};
+
+        auth0Client =  await auth0.createAuth0Client({
+            domain: config.domain,
+            clientId: config.clientId,
+            cacheLocation: 'localstorage'
+        });
+    }
+
+    const authentication = async () => {
+        await configureAuthClient();
+
+        const isAuthenticated = await auth0Client.isAuthenticated();
```

Comment:
**(question)** Does the method invoke the local cache check?

Reply:
I suppose it is

Reply:
There is no concrete information in documentation but in the source code the method checks cache